### ISSUE TITLE
3D roi_align and nms features request.

### ISSUE THREAD

**Issue by mjq93** [open-mmlab/mmcv#1092]
I want to implement 3D object detection based on medical volume data , but there is no 3D roi_align or  nms operator in the library. Are there any plans to add more 3D OPs?

**Comment by zhouzaida** [comment]
@ZwwWayne

**Comment by ZwwWayne** [comment]
Yes, in fact, for now, you can use [MMDetection3D](https://github.com/open-mmlab/mmdetection3d), which implements many 3D operators and support many 3D detection algorithms.

**Comment by mjq93** [comment]
> Yes, we do have plans but it takes some time. In fact, for now, you can use [MMDetection3D](https://github.com/open-mmlab/mmdetection3d), which implements many 3D operators and support many 3D detection algorithms.

@ZwwWayne Thank you for your patient reply. I have checked the MMDetection3D library carefully, but I did not find what I wanted. What I want is some ops  for 3D volume data , such as 3D ROI_Align and 3D ROI_Poolling, for which input shape is (B, C, D, H, W). Do you have any suggestions?

**Comment by puppy2000** [comment]
> > Yes, we do have plans but it takes some time. In fact, for now, you can use [MMDetection3D](https://github.com/open-mmlab/mmdetection3d), which implements many 3D operators and support many 3D detection algorithms.
> 
> @ZwwWayne Thank you for your patient reply. I have checked the MMDetection3D library carefully, but I did not find what I wanted. What I want is some ops for 3D volume data , such as 3D ROI_Align and 3D ROI_Poolling, for which input shape is (B, C, D, H, W). Do you have any suggestions?

So do you find any solutions?I have the same questions
